src/IssueViz/Editor/FilteringTaggerBase.cs

```diff
@@ -138,18 +138,11 @@ public IEnumerable<ITagSpan<TTagType>> GetTags(NormalizedSnapshotSpanCollection
 
             var filteredTagSpans = Filter(trackedTagSpans);
 
-            var currentSnapshot = GetSnapshot();
             var callerSnapshot = spans[0].Snapshot;
 
-            if (callerSnapshot.Version != currentSnapshot.Version)
-            {
-                Debug.Fail($"Expecting the tags to reference the correct snapshot. Caller snapshot version: {callerSnapshot.Version}, tag snapshot version: {currentSnapshot.Version}");
-                yield break;
-            }
-
             foreach (var dataTagSpan in filteredTagSpans)
             {
-                var filterSpans = dataTagSpan.Span.GetSpans(currentSnapshot);
+                var filterSpans = dataTagSpan.Span.GetSpans(callerSnapshot);
```

Comment:
This change should be fine. The spans returned by the tag aggregator seem to a type of tracking span, so they should be translated correctly when using `callerShapshot`.

I'm surprised our view tagger is being asked for tags for a newer version of the snapshot than the one currently the view is pointing to; I would have expected the view to be pointing at the latest version. 
However, that is what is happening. Our tagger is being called by the VS OutliningManager in response to a buffer change event, and this is happening before the text view updates the snapshot it is pointing to.